SConstruct

```diff
@@ -298,6 +298,8 @@ if selected_platform in platform_list:
     CXXFLAGS = env.get('CXXFLAGS', '')
     env['CXXFLAGS'] = ''
     env.Append(CXXFLAGS=str(CXXFLAGS).split())
+    if not env.msvc:
+        env.Append(CXXFLAGS="-std=c++11")
```

Comment:
While we'll do that Soon™ for real, it should be sufficient to do it for the `env_vlk_ldr` environment for now, no?

Reply:
Part of vulkan related includes (glslang) used in the "platform" have `nullptr`s, and "platform" doesn't have separate environment. This cause GCC 5 build to fail, newer compilers seems to have some of c+11 stuff enabled by default.

Reply:
Sounds like the ideal solution is to wait for after 3.2 is released and Godot upgrades C++ for 4.0.

Reply:
For the reference, this was added directly to the `master` branch in 5dae2ea777da5395cf1b1e9a8bc6abc93f6ae6bb so it's no longer included in this PR.